compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -441,6 +448,14 @@ void addMultiTilingExpertPassPipeline(OpPassManager &passManager,
     }
   }
 
+  {
+    nestedModulePM.addNestedPass<func::FuncOp>(
+        createFuseTensorPadWithConsumerPass());
+    nestedModulePM.addNestedPass<func::FuncOp>(
+        createConcretizePadResultShapePass());
+    nestedModulePM.addNestedPass<func::FuncOp>(createVectorizePadPass());
+  }
```

Comment:
extra scope is not needed here and below?

Reply:
good point!